Enterprise-CMCS/eAPD
# updated Activity Overview read-only for MMIS

Resolves #4553 

### Description
Updated the Activity read-only component. Pulled out the code for the Activity Overview into it's own component. Created components for HITECH Activity Overview and MMIS Activity Overview. The HITECH Activity Overview stays the same. The MMIS Activity Over was updated to match the Figma designs. Added tests and storybook for Activity Overview component.

I also added a knex template update for es6 style code.

Figma designs were updated:
https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=4717%3A20126&t=df84z13A4Rpoo6s8-1
https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=4271%3A19818&t=df84z13A4Rpoo6s8-1

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| a HITECH APD with data | viewing Activity Overview | the information displays as expected |
| a HITECH APD without data | viewing Activity Overview | the information displays as expected |
| a MMIS APD with data | viewing Activity Overview | the information displays as expected |
| a MMIS APD with most of the data | viewing Activity Overview | the information displays as expected |
| a MMIS APD without data | viewing Activity Overview | the information displays as expected |

### Steps to manually verify this change

1. Open an HITECH APD
2. Click on Export and Submit
3. Click on Continue to Review
4. Scroll down to Activities and verify that it looks the same

You will not be able to view the read-only version of MMIS in the app, but you can verify it in [storybook](https://www.chromatic.com/build?appId=61d5b948cf6f17003a12bf77&number=1071).

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
